bridgeifns.py

```diff
@@ -0,0 +1,348 @@
+        3025: f'3025: Failed to run interface_isolate payload on the enabled PodNet. Payload exited with status ',
+        3026: f'3026: Failed to connect to the enabled PodNet from the config file {config_file} for payload interface_ns:  ',
+        3027: f'3027: Failed to run interface_ns payload on the enabled PodNet. Payload exited with status ',
+        3028: f'3028: Failed to connect to the enabled PodNet from the config file {config_file} for payload interface_up:  ',
+        3029: f'3029: Failed to run interface_up payload on the enabled PodNet. Payload exited with status ',
+
+        3051: f'3051: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_check:  ',
+        3052: f'3052: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_add:  ',
+        3053: f'3053: Failed to run interface_add payload on the disabled PodNet. Payload exited with status ',
+        3054: f'3054: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_isolate:  ',
+        3055: f'3055: Failed to run interface_isolate payload on the disabled PodNet. Payload exited with status ',
+        3056: f'3056: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_ns:  ',
+        3057: f'3057: Failed to run interface_ns payload on the disabled PodNet. Payload exited with status ',
+        3058: f'3058: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_up:  ',
+        3059: f'3059: Failed to run interface_up payload on the disabled PodNet. Payload exited with status ',
+    }
+
+    # Default config_file if it is None
+    if config_file is None:
+        config_file = '/opt/robot/config.json'
+
+    status, config_data, msg = load_pod_config(config_file)
+    if not status:
+      if config_data['raw'] is None:
+          return False, msg
+      else:
+          return False, msg + "\nJSON dump of raw configuration:\n" + json.dumps(config_data['raw'],
+              indent=2,
+              sort_keys=True)
+    enabled = config_data['processed']['enabled']
+    disabled = config_data['processed']['disabled']
+
+
+    def run_podnet(podnet_node, prefix, successful_payloads):
+        rcc = SSHCommsWrapper(comms_ssh, podnet_node, 'robot')
+        fmt = PodnetErrorFormatter(
+            config_file,
+            podnet_node,
+            podnet_node == enabled,
+            {'payload_message': 'STDOUT', 'payload_error': 'STDERR'},
+            successful_payloads
+        )
+
+        payloads = {
+            'interface_check' : f'ip netns exec {namespace} ip link show {namespace}.{bridgename}',
+            'interface_add' : f'ip link add {bridgename}.{namespace} type veth peer name {namespace}.{bridgename}',
+            'interface_isolate' : f'ip link set dev {bridgename}.{namespace} master {bridgename}',
```

Comment:
```suggestion
            'interface_main' : f'ip link set dev {bridgename}.{namespace} master {bridgename}',
```
I feel the name is diverted so can you change it the suggestion please?